web/src/stream_edit.js

```diff
@@ -286,6 +287,29 @@ export function update_muting_rendering(sub) {
     $edit_container.find(".mute-note").toggleClass("hide-mute-note", !sub.is_muted);
 }
 
+function stream_notification_reset(e) {
+    const sub = get_sub_for_target(e.target);
+    if (!sub) {
+        blueslip.error("undefined sub in stream_notification_reset()");
+        return;
+    }
```

Comment:
Does this blueslip code serve a purpose? `get_sub_for_target` already throws an exception if it fails.

``` js
function get_sub_for_target(target) {                                                                   
    const stream_id = get_stream_id(target);                                                            
    if (!stream_id) {                                                                                   
        blueslip.error("Cannot find stream id for target");                                             
        return undefined;                                                                               
    }                                                                                                   
                                                                                                        
    const sub = sub_store.get(stream_id);                                                               
    if (!sub) {                                                                                         
        blueslip.error("get_sub_for_target() failed id lookup", {stream_id});                           
        return undefined;                                                                               
    }                                                                                                   
    return sub;                                                                                         
```

Reply:
Yeah you are right. Makes no sense to check it again, overlooked it.